jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -773,26 +738,25 @@ class DeploymentFlow extends React.Component {
       return;
     }
     return (
-      <juju.components.DeploymentSection
-        completed={status.completed}
-        disabled={status.disabled}
-        instance="deployment-services"
-        showCheck={true}
-        title={this._generateChangelogTitle()}>
-        <juju.components.DeploymentServices
-          acl={this.props.acl}
-          changesFilterByParent={this.props.changesFilterByParent}
-          charmsGetById={this.props.charmsGetById}
-          generateAllChangeDescriptions={
-            this.props.generateAllChangeDescriptions}
-          groupedChanges={this.props.groupedChanges}
-          listPlansForCharm={this.props.listPlansForCharm}
-          parseTermId={this._parseTermId.bind(this)}
-          servicesGetById={this.props.servicesGetById}
-          showChangelogs={this.state.showChangelogs}
-          showTerms={this.props.showTerms}
-          withPlans={this.props.withPlans} />
-      </juju.components.DeploymentSection>);
+      <div className="deployment-services">
+        <juju.components.AccordionSection
```

Comment:
In order to default these to collapsed, convert from the deploymentSection wrapper component to the AccordionSection component.